Octokit/Models/Response/Installation.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents an application installation.
+    /// </summary>
+    /// <remarks>
+    /// For more information see https://developer.github.com/v3/apps/#find-installations
+    /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class Installation
+    {
+        public Installation() { }
+
+        public Installation(int id, User account,
+            string accessTokenUrl, string repositoriesUrl, string htmlUrl,
+            int appId, int targetId, AccountType targetType,
+            string singleFileName, string repositorySelection)
+        {
+            Id = id;
+            Account = account;
+            AccessTokensUrl = accessTokenUrl;
+            RepositoriesUrl = repositoriesUrl;
+            HtmlUrl = htmlUrl;
+            AppId = appId;
+            TargetId = targetId;
+            TargetType = targetType;
+            SingleFileName = singleFileName;
+            RepositorySelection = repositorySelection;
+        }
+
+        public int Id { get; protected set; }
+        public User Account { get; protected set; }
+        public string AccessTokensUrl { get; protected set; }
+        public string RepositoriesUrl { get; protected set; }
+        public string HtmlUrl { get; protected set; }
+
+        public int AppId { get; protected set; }
+        public int TargetId { get; protected set; }
+        public StringEnum<AccountType> TargetType { get; protected set; }
+        // TODO - add permissions
```

Comment:
are these TODO comments going to be actioned in this PR?

Reply:
Not at the moment. They are merely placeholders so I won't forget where I need to add them.